src/components/NcAppContent/NcAppContent.vue

```diff
@@ -268,6 +279,24 @@ export default {
 				},
 			}
 		},
+
+		realPageTitle() {
+			if (this.pageTitle) {
+				return this.pageTitle
+			}
+			if (this.pageHeading) {
+				return t('{view} - {app} - {product}', { view: this.pageHeading, app: appName, product: productName })
```

Comment:
Does `{} - {} - {}` require translation? At least on server, it is not.

Reply:
Anyway, the server has it without translation now.

Reply:
It works fine when all the words are already translated and in RTL, space and dash doesn't change the direction.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/02896bf7-e385-4fc5-8138-2b80ca4b5d97)